backend/src/auth/authTotp.service.ts

```diff
@@ -1,64 +1,61 @@
-import {
-  BadRequestException,
-  ForbiddenException,
-  Injectable,
-  UnauthorizedException,
-} from "@nestjs/common";
+import { BadRequestException, ForbiddenException, Injectable, UnauthorizedException, } from "@nestjs/common";
 import { User } from "@prisma/client";
 import * as argon from "argon2";
 import { authenticator, totp } from "otplib";
-import * as qrcode from "qrcode-svg";
```

Comment:
There is a runtime error if you use `import QRCode from "qrcode-svg";` instead of `import * as qrcode from "qrcode-svg";`

I don't exactly know why this doesn't work in NestJS but in other frameworks your solution would work.

Reply:
Oh, so wield. I was warned by my IDE that constructors usually start with an uppercase, so I auto-fix it with my IDE.